coreos-cloudinit.go

```diff
@@ -72,7 +73,8 @@ func init() {
 	flag.BoolVar(&flags.printVersion, "version", false, "Print the version and exit")
 	flag.BoolVar(&flags.ignoreFailure, "ignore-failure", false, "Exits with 0 status in the event of malformed input from user-data")
 	flag.StringVar(&flags.sources.file, "from-file", "", "Read user-data from provided file")
-	flag.StringVar(&flags.sources.configDrive, "from-configdrive", "", "Read data from provided cloud-drive directory")
+	flag.StringVar(&flags.sources.configDrive, "from-configdrive", "", "[DEPRECATED - Use -from-openstack-configdrive] Read data from provided cloud-drive directory")
```

Comment:
Is it going to be feasible to remove this flag at some point? Seems like you could just leave this as-is and introduce other flags as necessary when they emerge?